### ISSUE TITLE
Wrong german translation of information on that page

### ISSUE THREAD

**Issue by Niko-O** [MicrosoftDocs/WSL#1701]
### Documentation Issue

This page contains a pink "Note" box containing this text:

> WSL 2 will work with [VMware 15.5.5+](https://blogs.vmware.com/workstation/2020/05/vmware-workstation-now-supports-hyper-v-mode.html) and [VirtualBox 6+](https://www.virtualbox.org/wiki/Changelog-6.0). Learn more in our [FAQs.](https://learn.microsoft.com/en-us/windows/wsl/faq#will-i-be-able-to-run-wsl-2-and-other-3rd-party-virtualization-tools-such-as-vmware--or-virtualbox-)

The text in this note on the german version of this page is this:

> Die WSL 2 verwendet [VMware 15.5.5+](https://blogs.vmware.com/workstation/2020/05/vmware-workstation-now-supports-hyper-v-mode.html) und [VirtualBox 6+](https://www.virtualbox.org/wiki/Changelog-6.0). Weitere Informationen finden Sie in den [häufig gestellten Fragen](https://learn.microsoft.com/de-de/windows/wsl/faq#will-i-be-able-to-run-wsl-2-and-other-3rd-party-virtualization-tools-such-as-vmware--or-virtualbox-).

Which literally translates to "WSL 2 **uses** VMware 15.5.5+ and VirtualBox 6+"!  (i.e. "uses" instead of "works with")
This is just flat-out wrong information, and it's extremely confusing.  

### Link to documentation page

https://learn.microsoft.com/de-de/windows/wsl/compare-versions
https://learn.microsoft.com/en-us/windows/wsl/compare-versions

### Suggested Improvements

Please fix the german translation.

This also alludes to a bigger problem: It's quite difficult to escape translation hell on Microsoft websites. This is especially true for machine-translated articles, because those are basically unreadable. (There are also some indicators that this article is machine-translated, but that's ususally indicated with a pink banner at the top, which isn't the case here, so it might just be some weirdness.) E.g. when I find articles on Google (german), the results tend to link to german articles, which tend to be machine-translated. To get to the english article, I have to painstakingly replace the "de-de" part in the URL with "en-us". This is so frustrating that people have even created browser extensions to automatically redirect.

Please consider making it easier to stick to the english articles. Ideally I'd just immediately get redirected to the english article if I open a german article. But even just a clearly visible link at the top would be helpful.

**Comment by mattwojo** [comment]
Unfortunately we currently have to rely on machine translation, but you can set the Learn content to display only in English using the language selector: https://learn.microsoft.com/en-us/locale/?target=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2F

**Comment by Niko-O** [comment]
> you can set the Learn content to display only in English using the language selector

It was already set to English. I clicked it again just to be sure. Whenever I open a link to the german version of an article, it still shows up in german.

I did find this link, though:
![grafik](https://github.com/MicrosoftDocs/WSL/assets/4943909/57b23e7f-1f91-46ea-8b85-daeb111e37dc)
It redirects to the english version. But it's still annoying that I have to click it every time I get linked to a non-english page.
